src/plugins/workflow_designer/src/cmdline/WorkflowCMDLineTasks.cpp

```diff
@@ -200,6 +223,11 @@ void WorkflowRunFromCMDLineBase::run() {
     reportFile.write(workflowRunTask->generateReport().toLocal8Bit());
 }
 
+QString WorkflowRunFromCMDLineBase::generateReport() const {
+    int i = 0;
```

Comment:
remove i=0

Reply:
`generateReport()` has been removed, it's been using for some debug stuff

Reply:
OK